.github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+This PR introduces a new <!-- insert new version here --> release in tezos-packaging.
+
+<!-- Depending on the type of the release some of the points below can be ommited.
+
+E.g. in case we introduce a new changes to our native packages, it's not necessary to bump
+used Tezos sources, create a new release in this repository and update brew formulas.
+
+Note that opam-repository is usually updated with some delay, so CI may fail
+due to lack of updated opam Tezos packages. In such case feel free to comment
+out native-packaging-related CI steps from the pipeline until the opam-repository
+is updated.
+-->
+
+<!-- List issues which are going to be resolved in the new packages from this
+release. Write 'None' if there are no related issues.
+
+For example:
+Follows #202, #210
+-->
+
+Follows #
+
+### Changes related to the creation of a release (conditional)
+
+- [ ] I updated Tezos sources and opam-repository (if needed) revisions in [sources.json](../../tree/master/nix/nix/sources.json).
+- [ ] I removed old bottles hashes from the brew formulas in [Formula directory](../../tree/master/Formula).
+- [ ] I updated `url :tag` and `version`s in brew formulas..
+- [ ] I updated release number in [meta.json](../../tree/master/meta.json).
+
+#### In case the new Tezos release provides a new protocol and corresponding testnet
+
+- [ ] I supported new protocol in [protocols.json](../../tree/master/protocols.json).
+- [ ] I supported new protocol and testnet in native packaging.
+- [ ] I supported new protocol and testnet in brew formulas.
+- [ ] I added tests for the new protocol and testnet.
+
+### Things to do after this release PR is merged
+
+<!--
+Some of the changes are done after the new release is created, consider checking
+the merged release PR with this template and follow unfinished points.
+-->
```

Comment:
```suggestion
Some of the changes are done after the new release is created, consider following 
and checking unfinished points in the merged release PR using this template.
-->
```
It took me a minute to parse otherwise.